.gitignore

```diff
@@ -1 +1,4 @@
-node_modules/
\ No newline at end of file
+node_modules/
+Public/Helpers/
```

Comment:
Ignore the entire Public folder maybe? Also just a nitpick you can name it `dist` or `compiled` or `bin` or something else that is widely used in different frontend frameworks